code/controllers/subsystem/events.dm

```diff
@@ -64,7 +64,16 @@ SUBSYSTEM_DEF(events)
 	// Only alive, non-AFK human players count towards this.
 
 	var/sum_of_weights = 0
-	for(var/datum/round_event_control/E in control)
+
+	//NSV13 - are we using our event list or the full one?
+	var/obj/structure/overmap/mainship = SSstar_system.find_main_overmap()
+	var/list/possible_events
+	if(mainship.current_system && length(mainship.current_system.possible_events) && prob(50))
+		possible_events = mainship.current_system.possible_events
```

Comment:
This also makes it really easy to trigger events for other ships as well when we change events to be compatible with that (thinking abt. PvP or boarding ships)